src/gwt/src/org/rstudio/studio/client/workbench/views/terminal/events/TerminalSessionListEvent.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * TerminalSessionListEvent.java
```

Comment:
Is this event ever fired on anything other than session init? If not you could probably just piggyback on the existing `SessionInitEvent` as it already carries the set of processes. (No need to change if the existing mechanism has other benefits or just seems more straightforward to you)